enterprise/backend/src/metabase_enterprise/caching/api.clj

```diff
@@ -0,0 +1,118 @@
+(ns metabase-enterprise.caching.api
+  (:require
+   [compojure.core :refer [GET]]
+   [metabase.api.common :as api]
+   [metabase.api.common.validation :as validation]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.events :as events]
+   [metabase.util :as u]
+   [metabase.util.cron :as u.cron]
+   [metabase.util.i18n :refer [tru]]
+   [metabase.util.malli.schema :as ms]
+   [toucan2.core :as t2]))
+
+(defn- assert-valid-models [model ids]
+  (if (= model "root")
+    (when-not (zero? (first ids))
```

Comment:
I mean I could, but OTOH this won't make any real difference. I only check it there so that frontend will know that 0 will come back as a model id for root. I could've hidden that, but that would've made schema and code slightly more complicated.

Reply:
What's the downside to doing `(when-not (= [0] ids) ...` instead?